src/System.Net.Requests/tests/HttpWebRequestTest_1.7.cs

```diff
@@ -0,0 +1,90 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Please don't name files with "_1.7".  There is already an effort to default to netstandard17 for the repo anyways.